### PR TITLE
Us060 filter respondents by go live date

### PR DESCRIPTION
**Pull request by andrewmil**
Previously we were using the print date not the go live event date to filter surveys

Added controller for retrieving specific collection exercise events
Now filter on the go_live event
Updated/Added a test accordingly

Integration tests will only pass using [this branch of rm-tools](https://github.com/ONSdigital/rm-tools/pull/13)

[Trello](https://trello.com/c/Bp82qykc/74-us060-ext-filter-a-respondents-view-of-ce-based-on-ce-go-live-date-m-15pts-3d)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/200933471/US060+Filter+a+respondents+view+of+a+CE+based+on+the+CE+Go-Live+Date+event)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US060: EXT Filter a respondents View of CE based on CE Go-Live Date (M = 15pts = 3d)](https://trello.com/c/Bp82qykc/74-us060-ext-filter-a-respondents-view-of-ce-based-on-ce-go-live-date-m-15pts-3d)

File: frontstage_api/controllers/case_controller.py [hunk 1]
```diff
@@ -124,6 +125,7 @@ def build_full_case_data(case):
         "status": status,
         "collection_instrument_type": collection_instrument['type'],
         "collection_instrument_size": collection_instrument['len'],
+        'go_live': collection_exercise_go_live
```
Comment (tej99) [inline frontstage_api/controllers/case_controller.py]: Can we use double quotes here to keep it consistent

File: .pytest_cache/v/cache/lastfailed [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (tej99) [inline .pytest_cache/v/cache/lastfailed]: Need to add .pytest_cache into gitignore

File: frontstage_api/controllers/collection_exercise_controller.py [hunk 3]
```diff
@@ -37,3 +38,18 @@ def get_collection_exercise_events(collection_exercise_id):
...
+
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        raise ApiError(url=url, status_code=response.status_code,
+                       description='Failed to retrieve collection exercise event',
+                       collection_exercise_id=collection_exercise_id, tag=tag)
+
+    logger.debug('Successfully retrieved collection exercise event', collection_exercise_id=collection_exercise_id, tag=tag)
+    return json.loads(response.text)
```
Comment (tej99) [inline frontstage_api/controllers/collection_exercise_controller.py]: response.json() ?

Comment (dcdarrell9) [reply frontstage_api/controllers/collection_exercise_controller.py]: The function above `def get_collection_exercise_events(collection_exercise_id):` uses 
`return json.loads(response.text)` so would be nice to have both the same

Comment (andrewmil) [reply frontstage_api/controllers/collection_exercise_controller.py]: In general we should be using `response.json()`  it's just cleaner. I didn't know about it for ages so we use the other way across the frontstages/backstages 🤦‍♂️.  I'll update these both

**Pull request merged**